puppet/modules/dgu_ckan/manifests/init.pp

```diff
@@ -584,7 +584,7 @@
   # I can replicate this on the command-line when logged in as vagrant and using sudo,
   # but it works if you do 'sudo su' or 'sudo su co' and then do 'sudo npm install'.
   exec { 'npm_deps_dgu':
-    command   => 'sudo npm install',
+    command   => 'sudo npm install --no-bin-links',
```

Comment:
It's not a windows things. My host machine is linux. Is it working with you guys?